tests/augmentation/test_augmentation.py

```diff
@@ -3509,6 +3510,66 @@ def test_same_on_batch(self, device, dtype):
         self.assert_close(output_tensor[0], output_tensor[1])
 
 
+class TestRandomChannelDropout(BaseTester):
+    def _get_expected(self, device, dtype):
+        return torch.tensor(
+            [
+                [
+                    [[1.0, 1.0, 1.0], [1.0, 1.0, 1.0], [1.0, 1.0, 1.0]],
+                    [[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]],
+                    [[1.0, 1.0, 1.0], [1.0, 1.0, 1.0], [1.0, 1.0, 1.0]],
+                ]
+            ],
+            device=device,
+            dtype=dtype,
+        )
+
+    def test_smoke(self, device, dtype):
+        torch.manual_seed(1)
+        input_tensor = torch.ones(1, 3, 3, 3, device=device, dtype=dtype)
+        expected = self._get_expected(device=device, dtype=dtype)
+        aug = RandomChannelDropout(p=1.0)
+        res = aug(input_tensor)
+        assert input_tensor.shape == res.shape
+        self.assert_close(res, expected, rtol=1e-4, atol=1e-4)
+
+    def test_exception(self, device, dtype):
+        num_drop_channels = 2.0
+        with pytest.raises(TypeError, match=f"`num_drop_channels` must be an int. Got: {type(num_drop_channels)}"):
+            RandomChannelDropout(num_drop_channels=num_drop_channels)
+
+        num_drop_channels = 5
+        with pytest.raises(
+            Exception,
+            match=f"Invalid value in `num_drop_channels`. Must be an int bewteen 1 and 3. Got: {num_drop_channels}",
+        ):
+            RandomChannelDropout(num_drop_channels=num_drop_channels)
+
+        fill_value = 2.0
+        with pytest.raises(
+            Exception, match=f"Invalid value in `fill_value`. Must be a float bewteen 0 and 1. Got: {fill_value}"
+        ):
+            RandomChannelDropout(fill_value=fill_value)
+
+        fill_value = 1
+        with pytest.raises(TypeError, match=f"`fill_value` must be a float. Got: {type(fill_value)}"):
+            RandomChannelDropout(fill_value=fill_value)
+
+    @pytest.mark.parametrize("channel_shape, batch_shape", [(3, 1), (3, 2), (3, 5)])
```

Comment:
```suggestion
    @pytest.mark.parametrize("channel_shape, batch_shape", [(3, 1), (1, 1), (3, 5)])
```

Reply:
Why this change?

In local tests are working fine (cpu).

Reply:
To have tests with other C size